CMakeLists.txt

```diff
@@ -20,6 +20,7 @@
 
 cmake_minimum_required(VERSION 3.4)
 include(FeatureSummary)
+include(GNUInstallDirs)
```

Comment:
Probably the block
```cmake
if(NOT DEFINED BINDIR)
  set(BINDIR "bin")
endif(NOT DEFINED BINDIR)
```
in line 401 should be removed now. Did you check that the paths provided by `GNUInstallDirs` are used for all install targets? I did a quick grep and did find nothing, but I only checked the `install` commands, not sure
if this effects rpaths and so on....

Reply:
Thanks! I overlooked that one. The `install()` commands have to be configured such that submodules get installed in the python `espressomd` folder, if not the new test in c43d057 will detect that (it searches for misplaced files in `CMAKE_INSTALL_PREFIX`). The GNU variables inherit from `CMAKE_INSTALL_PREFIX`, so we should use them as often as possible.